### PR TITLE
Penthouse pod rework plus new garden pod. 

### PR DESCRIPTION
**Pull request by shellspeed1**
penthouse pod update

## About The Pull Request
This PR reworks the penthouse survival pod, provides several general reworks to the powered survival pods, fixes an issue with the pod windows, and adds a new survival pod to replace the removed garden and kitchen the penthouse pod used to have.

New Penthouse Pod

 Screenshot (625)

New garden pod

Screenshot (627) 

## Why It's Good For The Game
Fixed the window x4 issue, makes the penthouse pod survival pod more suited to its intended survival pod, improves pod modular design.

## Changelog
:cl:
add: new garden survival pod added. . It can be bought from the mining vendor for 7500 points.
balance: To reflect the drop in usefulness of the penthouse pod, its price has been halved.
tweak: Penthouse survival pod redone, instead of a kitchen and garden it now features an office area for when a miner feels like writing a book, or shitposting in the station's modular computer chatroom.
fix: Stacked windows in survival pods fixed.
tweak: rtgs in survival pods now come prewired, I figured out a new trick woo.

/:cl:

### CODE REVIEW

**Comment by shellspeed1** [comment]
Still needs a bit of test merging, and I can probably remove the insulated gloves plus toolbox from the penthouse and garden pod. I'll probably throw it in the empty pod since it doesn't come with power by default.  Anyway I need to test these more later, its  5 am and I'm not fully here rn.

**Comment by shellspeed1** [comment]
After doing a quick run through I didn't see anything obvious that I missed issue with so I'm going to call these test merge ready since we are still doing a bunch of test merges last I checked.

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -57,7 +57,8 @@
 		new /datum/data/mining_equipment("Luxury Shelter Capsule",		/obj/item/survivalcapsule/luxury,									3000),
 		new /datum/data/mining_equipment("Luxury Bar Capsule",			/obj/item/survivalcapsule/luxury/elitebar,							10000),
 		new /datum/data/mining_equipment("Empty Capsule",				/obj/item/survivalcapsule/luxury/empty,								5000),
-		new /datum/data/mining_equipment("Penthouse Capsule",			/obj/item/survivalcapsule/luxury/penthouse,							15000),
+		new /datum/data/mining_equipment("Penthouse Capsule",			/obj/item/survivalcapsule/luxury/penthouse,							7500),
+		new /datum/data/mining_equipment("garden & kitchen Capsule",	/obj/item/survivalcapsule/luxury/garden,							7500),
```
Comment (SandPoot) [inline code/modules/mining/machine_vending.dm]: Should probably capitalize here

File: code/modules/mining/equipment/survival_pod.dm [hunk 2]
```diff
@@ -91,9 +91,13 @@
...
 /obj/item/survivalcapsule/luxury/penthouse
-	name = "penthouse bluespace sheler capsule"
-	desc = "The absolute pinnacle of luxury in terms of survival capsules. While exuberantly expensive it has everything needed to make a small home in lavaland."
+	name = "penthouse bluespace shelter capsule"
+	desc = "The absolute pinnacle of luxury in terms of survival capsules. While exuberantly expensive it has everything needed to survive in luxury."
 	template_id = "shelter_epsilon"
+/obj/item/survivalcapsule/luxury/garden
+	name = "garden & kitchen bluespace shelter capsule"
+	desc = "Everything someone needs to make a home cooked meal while surviving the depths of hell... or space."
+	template_id = "shelter_Zeta"
```
Comment (SandPoot) [inline code/modules/mining/equipment/survival_pod.dm]: Does this Z have to be capitalized?

Comment (shellspeed1) [reply code/modules/mining/equipment/survival_pod.dm]: Technically no, but it needs to be changed elsewhere. Went ahead and did that.

File: code/modules/mining/machine_vending.dm (hunk 1 above)
Comment (shellspeed1) [reply code/modules/mining/machine_vending.dm]: Done.

**Pull request merged**